domains/authorization/Authorization.API/API/Migrations/20241115125358_AddDefaultServiceProviderTermsWithUniqueConstraint.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace API.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddDefaultServiceProviderTermsWithUniqueConstraint : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.InsertData(
```

Comment:
I am kinda scared after last time about inserting data as part of migrations, since we saw it updated terms on all migrations in future.

Reply:
this time it's different. now it's only an migration